pom.xml

```diff
@@ -120,6 +122,26 @@
                 <artifactId>farao-glsk-document-api</artifactId>
                 <version>${farao.core.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.farao-community.farao</groupId>
+                <artifactId>farao-crac-creator-api</artifactId>
+                <version>${farao.core.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.rte-france.farao</groupId>
+                <artifactId>farao-data-fb-constraint</artifactId>
+                <version>${farao.rte.core.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.farao-community.farao</groupId>
+                <artifactId>gridcapa-rao-runner-spring-boot-starter</artifactId>
+                <version>${gridcapa.rao.runner.starter.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.farao-community.farao</groupId>
+                <artifactId>gridcapa-rao-runner-api</artifactId>
```

Comment:
Same remark, maybe this dependency should be moved to gridcapa-core-valid-app pom file ? It would be more logical I think

Reply:
same we need it here

Reply:
in fact in can be removed from both pom files